railties/lib/rails/test_unit/runner.rb

```diff
@@ -45,7 +45,7 @@ def load_tests(argv)
           patterns = extract_filters(argv)
 
           tests = Rake::FileList[patterns.any? ? patterns : "test/**/*_test.rb"]
-          tests.exclude("test/system/**/*", "test/dummy/**/*") if patterns.empty?
+          tests.exclude(ActiveSupport.excluded_test_paths) if patterns.empty?
```

Comment:
Can't remember if there was a discussion back when we added this, but curious what it would look like if we flipped it? Such that we only recognize `test/models`, `test/controllers` by default? Or perhaps only run tests for corresponding directories in the app by default? That would set us up with the argument that `rails test` only runs `app/` tests by default (and others' can be run with `rails test test/system` etc.).

Also, doesn't seem right that we just reference `ActiveSupport` here.

Reply:
> Or perhaps only run tests for corresponding directories in the app by default? That would set us up with the argument that rails test only runs app/ tests by default (and others' can be run with rails test test/system etc.).

I like this idea. But couldn't that be a breaking change for a lot existing code (for people not following those conventions with test folders)?

Reply:
> Also, doesn't seem right that we just reference ActiveSupport here.

@kaspth maybe better moving the constant to `Rails::TestUnit::Runner` and initialize it in `TestUnitRailtie`?